source/npm-util.js

```diff
@@ -61,3 +62,11 @@ exports.prereleaseTags = async packageName => {
 
 	return tags;
 };
+
+exports.verifyRecentNpmVersion = async () => {
+	const versions = JSON.parse(await execa.stdout('npm', ['version', '--json']));
+
+	if (version(versions.npm).satisfies('<6.8.0')) {
```

Comment:
to be easier to maintain, I suppose it might be worth to add a requirements, something like "engines" in "package.json":
```js
{
  "npm": ">=6.8.0",
  "git": ">=..."
}
```

it just come to me when reviewing the change, but surely it doesn't have to be in this PR. 😄 thoughts?

Reply:
I just noticed that [the `engines` field does support setting the required npm version](https://docs.npmjs.com/files/package.json#engines), and so we should set it as you recommended. However, setting the git version would be ignored, and so I don't think there's any reason to do that.